js/apps/admin-ui/src/realm-settings/TokensTab.tsx

```diff
@@ -77,9 +84,30 @@ export const RealmSettingsTokensTab = ({
     defaultValue: false,
   });
 
-  return (
-    <PageSection variant="light">
-      <FormPanel title={t("general")} className="kc-sso-session-template">
+  // Hydrate form values from realm attributes for OID4VCI
+  useEffect(() => {
```

Comment:
Why handle setting these properties in a side effect that then populates a different field? Can you not use the values directly? If they are the wrong type, perhaps the data loader would be a better place to convert them first (e.g `setupForm()` in `realm-settings/RealmSettingsTabs.tsx`), that way the conversion logic is co-located with the `save()` function that converts it back.